ui/src/app/applications/components/applications-list/applications-list.scss

```diff
@@ -67,77 +67,45 @@
         }
     }
 
-    $filters-container-collapsed-height: 3em;
-    $filters-container-box-shadow: 1px 1px 3px $argo-color-gray-5;
-
-    &__filters-container {
```

Comment:
On a small screen filters take quite a lot of space even if fully collapsed:

![image](https://user-images.githubusercontent.com/426437/116455101-89866d00-a815-11eb-9aac-5db6cd8bc6c5.png)

Before it was possible to hide them completely. Can we add collapsable container?